__tests__/seed.js

```diff
@@ -0,0 +1,58 @@
+import { knex } from 'knex';
+import log from 'loglevel';
+const uuid = require('uuid');
+
+const connection = process.env.DATABASE_URL;
+
+!connection && log.warn('env var DATABASE_URL not set');
+
+const knexConfig = {
+  client: 'pg',
+  // debug: process.env.NODE_LOG_LEVEL === 'debug',
+  debug: true,
+  connection,
+  pool: { min: 0, max: 10 },
+};
+
+const dataRawCaptureFeature = [
+  {
+    id: uuid.v4(),
+    lat: '41.50414585511928',
+    lon: '-75.66275380279951',
+    location: '0101000020E6100000B514ED8E6AEA52C05D13F4D987C04440',
+    field_user_id: 5127,
+    field_username: 'test',
+    created_at: new Date().toUTCString(),
+    updated_at: new Date().toUTCString(),
+  },
+  {
+    id: uuid.v4(),
+    lat: '57.57641356164619',
+    lon: '-113.11416324692146',
+    location: '0101000020E6100000B4FB5C734E475CC0E21E6AEBC7C94C40',
+    field_user_id: 5127,
+    field_username: 'test',
+    created_at: new Date().toUTCString(),
+    updated_at: new Date().toUTCString(),
+  },
+];
+
+async function seed() {
+  knexConfig.searchPath = [process.env.DATABASE_SCHEMA, 'webmap'];
+  const serverCon = knex(knexConfig);
+  const response = await serverCon.transaction(async (trx) => {
```

Comment:
It would be good to rename the existing `raw_capture_feature` table to another name (say `temp_raw_capture_feature`), then fill a newly created table with your seed data and name it to `raw_capture_feature` for your test. The test can proceed to completion, then you can delete the created table and rename `temp_raw_capture_feature` back to `raw_capture_feature`. This way will avoid clearing actual test data once ETL scripts begin to populate `raw_capture_feature`, and you won't have to rewrite leaderboard tests to account for the the new data in the table. You can use this knex [documentation](https://knexjs.org/#Schema-renameTable).

Reply:
Oh, yes, I know this is annoying, but, just don't touch the tables in `public` , you all can freely delete data under `webmap` schema, that's totally fine, because we didn't use that table for web map presentation.

And the next step would be use dedicated database for test, so we can do anything safely, there already are good practice in other our microservice for this, so we can learn what they are doing

Reply:
For me, I run a docker image of postgresql, and run db-migrate to set up the database, then I can run the test fastly, for other API service, like: earnings-api